test/utils.sh

```diff
@@ -628,9 +628,14 @@ extract_differential_fbc_metadata() {
     unique_index=$(echo "$package_result_index" | jq -s "flatten(1) | unique")
   fi
 
+  # Convert multiple JSON arrays on separate lines to a single valid JSON array
+  # Failure to do so will result in "jq: invalid JSON text passed to --argjson"
+  unique_index_json=$(echo "$unique_index" | jq -s 'add')
+  unique_fbc_json=$(echo "$unique_fbc" | jq -s 'add')
```

Comment:
Does this always need to happen or just for the bundle images? The related images function above uses `flatten(1)` for a similar reason. Can we perform the same action for both extraction operations to further simplify the code?

Reply:
sure, let me try that

Reply:
@arewm I added the jq "add" operation as a part of the extraction operation under the "unique_bundles". Also, while testing these changes for "related_images" extraction operation I discovered another issue, namely, error "/usr/bin/jq: Argument list too long". The parameter containing related images that we pass to jq --argjson is too long. To fix that, I saved the variables into temp files. PTAL

Reply:
Aha, that is interesting! Thanks for fixing that as well.